src/dnd-list/styled-components.js

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index.js';
+import type {ThemeT} from '../styles/index.js';
+import type {SharedStylePropsArgT} from './types.js';
+type StylePropsT = SharedStylePropsArgT & {
+  $theme: ThemeT,
+};
+
+/**
+ * Main component container element
+ */
+export const Root = styled('div', ({$theme}: StylePropsT) => {
+  return {
+    color: $theme.colors.mono1000,
+    width: '100%',
+  };
+});
+Root.displayName = 'StyledRoot';
+
+export const List = styled('ul', ({$isDragged}: StylePropsT) => {
+  return {
+    padding: 0,
+    cursor: $isDragged ? 'grabbing' : null,
+  };
+});
+List.displayName = 'StyledList';
+
+export const Item = styled(
```

Comment:
Great catch! It happens since the lifted item is portaled to the `document.body` and lost some cascaded styles - just added explicit styling for the font. 

Also, I replaced all `em` values with `$theme.space` since those were introducing a similar problem (lifted item had different spacing).